app/views/health/careplans/_careplan_form.haml

```diff
@@ -42,12 +42,11 @@
 
 
 
-= javascript_include_tag 'moment.min.js'
 = content_for :page_js do
   :javascript
 
     $('.jInitialDate').on('change', function(e) {
-      var initial_date = moment($(this).val()).format("MMM DD, YYYY");
-      var review_date = moment(initial_date).add(6, 'months');
-      $('.jReviewDate').val(review_date.format("MMM DD, YYYY"));
+      const initial_date = moment($(this).val(), "MMM DD, YYYY")
+      const review_date = initial_date.add(12, 'months');
```

Comment:
Is this a change in the api that goes from 6 -> 12 months?  or something else?

Reply:
It was a bug where it only added 6 months instead of 12. We updated it from 6->12 at some point but forgot to change it here https://github.com/greenriver/hmis-warehouse/commit/5957da7f47b78d0bb64d838196ce1a0238dfa0e3#diff-02342bb1f44b901c0f9e519c6a67144e876645eff7355516fd6bf80f241ae8d2

Reply:
I just happened across this when I was removing unnecessary direct imports of `moment.min.js`

Reply:
Good catch, thanks!